credentials/solid/credential.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": [ "https://www.w3.org/2018/credentials/v1", "https://purl.imsglobal.org/spec/ob/v3p0/context.json", "https://w3id.org/security/suites/ed25519-2020/v1" ],
+  "id": "urn:uuid:951b475e-b795-43bc-ba8f-a2d01efd2eb1",
+  "type": [ "VerifiableCredential", "OpenBadgeCredential" ],
+  "issuer": {
+    "id": "did:key:z6Mkr3oy4PPP5cBpdgvmz1mqqPWkVrzzkEq2btA1exN9YiVX",
+    "type": "Profile",
+    "name": "SOLID, LLC.\r\nEndorsed by The Manufacturing Institute",
```

Comment:
Recommend removing `\r\`. These characters may be rendered as-is in some wallets.